plothist/plotters.py

```diff
@@ -146,28 +168,28 @@ def make_hist(data, bins=50, range=[None, None], weights=1):
         filled histogram
     """
 
-    binning = _format_binning(data, range, bins)
+    axis = create_axis(data, bins, range)
 
     if weights is None:
         storage = bh.storage.Double()
     else:
         storage = bh.storage.Weight()
 
-    h = bh.Histogram(bh.axis.Regular(*binning), storage=storage)
+    h = bh.Histogram(axis, storage=storage)
     h.fill(data, weight=weights, threads=0)
 
     # Check what proportion of the data is in the underflow and overflow bins
     range_coverage = h.values().sum() / h.values(flow=True).sum()
     # Issue a warning in more than 1% of the data is outside of the binning range
     if range_coverage < 0.99:
         warnings.warn(
-            f"Only {100*range_coverage:.2f}% of data contained in the binning range ({binning[1]}, {binning[2]})."
+            f"Only {100*range_coverage:.2f}% of data contained in the binning range ({axis.edges[0]}, {axis.edges[-1]})."
```

Comment:
Why [-1] except of [2]?

Reply:
Because this is the list of all bin edges, including internal edges.